packages/iov-ethereum/src/ethereumcodec.ts

```diff
@@ -92,6 +94,8 @@ export const ethereumCodec: TxCodec = {
       otherSignatures: [],
     };
   },
-  keyToAddress: keyToAddress,
+  identityToAddress: (identity: PublicIdentity): Address => {
```

Comment:
This is same on all networks? interesting

Reply:
Jap. This interface helps us understanding blockchains :)